### DISCUSSION TITLE
DEPLOYING TO PRODUCTION

### DISCUSSION THREAD

**Discussion by fkaris** [filamentphp/filament#4000] [Help]
hello experts, my laravel app is 1.4gb after using the filament admin for s simple page, question is how can i shrink the application?

**Comment by themahdavi** [comment]
1.4G just by adding filament?😳

**Comment by danharrin** [comment]
You don't need to shrink anything... maybe your app is 1.4gb with Composer dependencies committed to Git? Those should be installed on your server, not uploaded
